firebase-vscode/webviews/SidebarApp.tsx

```diff
@@ -88,16 +90,26 @@ function EmulatorsPanel() {
     }
     return runningPanel;
   }
+  console.log(emulatorsRunningInfo.value);
 
-  return emulatorsRunningInfo.value?.infos ? (
+  return (emulatorsRunningInfo.value?.infos && emulatorsRunningInfo.value?.status === "running") ? (
     <Emulators emulatorInfo={emulatorsRunningInfo.value.infos!} />
   ) : (
-    <VSCodeButton
-      appearance="secondary"
-      onClick={() => broker.send("runStartEmulators")}
-    >
-      Start emulators
-    </VSCodeButton>
+    <>
+      <VSCodeButton
+        appearance="secondary"
+        onClick={() => broker.send("runStartEmulators")}
+      >
+        Start emulators
+      </VSCodeButton>
+      <Spacer size="xsmall" />
+      <Label level={3}>
+        See also:{" "}
+        <a href="https://firebase.google.com/docs/emulator-suite">
```

Comment:
VsCodeLink?

Reply:
can do another pass on all the "a" links later